### PR TITLE
feat: add image embedding category model

### PR DESCRIPTION
**Pull request by raphael0202**
Add categorization model with image embedding as input.

Also:
- Add label files (labels.txt) to repository. These files never change and it makes deployment easier.
- Add pbtxt config for clip model to increase max_batch_size to 32 (otherwise, it's limited to 4 by default)

### CODE REVIEW

**Comment by raphael0202** [comment]
Merging to test new model in production, review will be done afterwards.

**Review by alexgarel — COMMENTED** [review]
Great !

Just a bunch of comment.

File: robotoff/prediction/category/neural/keras_category_classifier_3_0/__init__.py [hunk 1]
```diff
@@ -3,17 +3,65 @@
...
+
+def generate_image_embeddings(product: JSONType, stub) -> Optional[np.ndarray]:
+    """Generate image embeddings using CLIP model for the `MAX_IMAGE_EMBEDDING`
+    most recent images.
+
+    :param product: product data
+    :param stub: the triton inference stub to use
+    :return: None if no image was available or a numpy array of shape
+        (num_images, IMAGE_EMBEDDING_DIM)
+    """
```
[the remark this answers is not in the document]
Comment (raphael0202) [reply robotoff/prediction/category/neural/keras_category_classifier_3_0/__init__.py:35]: Indeed that could have been a good strategy. But as the training was done by using the most recent images, I would rather keep it this way.

File: robotoff/prediction/category/neural/keras_category_classifier_3_0/__init__.py [hunk 2]
```diff
@@ -193,4 +281,23 @@ def build_triton_request(
...
+        value = inputs["image_embeddings"]
+        request.raw_input_contents.extend([value.tobytes()])
+
+        image_embeddings_mask_input = service_pb2.ModelInferRequest().InferInputTensor()
+        image_embeddings_mask_input.name = "image_embeddings_mask"
+        image_embeddings_mask_input.datatype = "FP32"
+        image_embeddings_mask_input.shape.extend([1, MAX_IMAGE_EMBEDDING])
+        request.inputs.extend([image_embeddings_mask_input])
+        value = inputs["image_embeddings_mask"]
+        request.raw_input_contents.extend([value.tobytes()])
```
[the remark this answers is not in the document]
Comment (raphael0202) [reply robotoff/prediction/category/neural/keras_category_classifier_3_0/__init__.py:301]: I've added a few comments on new PR!

**Pull request merged**